### ISSUE TITLE
Is there a way to only authorize some private organizations, but not others?

### ISSUE THREAD

**Issue by jackiekazil** [rauhryan/huboard#506]
Is there a way to only authorize private repos from some organizations, but not others? 
(I don't want to authorize repos for all the orgs that I am a part of, but I can't figure out how to do that.)

**Comment by discorick** [comment]
The authorization for private repos belonging to orgs goes only as far as HuBoard "seeing" that you are a collaborator for them. The organizations themselves are responsible for whether they allow integrators (like HuBoard) to pull their issues. 

Additionally if you try to visit a repo belonging to an organization that has no active HuBoard subscription, you will encounter a 404. Does that clear things up?

**Comment by jackiekazil** [comment]
Yes, but as a user who is collaborator with 15 organizations (some hidden), I don't want huboard to request all the administrators of those organizations. I would like to select which organizations that huboard privately integrates with. Make sense?

**Comment by dahlbyk** [comment]
> I would like to select which organizations that huboard privately integrates with. Make sense?

This absolutely makes sense, but is not a capability offered by GitHub's access controls — our options are literally "public" or "public + private". We get this feedback often, and will keep working with GitHub to improve the granularity with which you can allow access.
